aws-rds-dbinstance/aws-rds-dbinstance.json

```diff
@@ -312,6 +312,18 @@
       "description": "The open mode of an Oracle read replica. The default is open-read-only.",
       "type": "string"
     },
+    "RestoreTime": {
+      "description": "The date and time to restore from.",
+      "type": "string"
```

Comment:
have you considered using `"format": "date-time"` in the schema? According to Json Schema it will get transformed to `java.util.Date`: https://github.com/joelittlejohn/jsonschema2pojo/wiki/Reference